packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -93,27 +93,27 @@ class _DebuggingControlsState extends State<DebuggingControls>
     final isSystemIsolate = controller.isSystemIsolate;
     return RoundedButtonGroup(
```

Comment:
Now that we only have one item in `items`, I don't think we need the `RoundedButtonGroup` (since its intended for a group of buttons). I think we probably want to use `DevToolButton` here (though we might need to add a `rounded` option which would match the UI of `RoundedButtonGroup). Adding @kenzieschmoll if she has other thoughts!

Reply:
+1 we can use `DevToolsButton` instead. One clean up could be to change this method to return a ternary:

` return isPaused ? DevToolsButton(...) : DevToolsButton(...);`;